src/Server/middleware/userRequiredMiddleware.ts

```diff
@@ -7,6 +7,7 @@ const USER_REQUIRED_ROUTES = [
   "/settings/payments",
   "/settings/purchases(.*)",
   "/settings/shipping",
+  "/notifications",
```

Comment:
if a user who is not logged in tries to open the page, we will redirect to the log in page

Reply:
alphabetical pls 🙏

Reply:
Done